diff --git a/node_modules/react-native-gesture-handler/lib/typescript/components/Swipeable.d.ts b/node_modules/react-native-gesture-handler/lib/typescript/components/Swipeable.d.ts
index 53f7145..24fd839 100644
--- a/node_modules/react-native-gesture-handler/lib/typescript/components/Swipeable.d.ts
+++ b/node_modules/react-native-gesture-handler/lib/typescript/components/Swipeable.d.ts
@@ -106,12 +106,12 @@ export interface SwipeableProps extends Pick<PanGestureHandlerProps, SwipeableEx
      * Style object for the container (`Animated.View`), for example to override
      * `overflow: 'hidden'`.
      */
-    containerStyle?: StyleProp<ViewStyle>;
+    containerStyle?: Animated.WithAnimatedObject<ViewStyle> | Animated.WithAnimatedArray<ViewStyle>;
     /**
      * Style object for the children container (`Animated.View`), for example to
      * apply `flex: 1`
      */
-    childrenContainerStyle?: StyleProp<ViewStyle>;
+    childrenContainerStyle?: Animated.WithAnimatedObject<ViewStyle> | Animated.WithAnimatedArray<ViewStyle>;
 }
 declare type SwipeableState = {
     dragX: Animated.Value;
diff --git a/node_modules/react-native-gesture-handler/src/components/Swipeable.tsx b/node_modules/react-native-gesture-handler/src/components/Swipeable.tsx
index c654c66..0ff8f71 100644
--- a/node_modules/react-native-gesture-handler/src/components/Swipeable.tsx
+++ b/node_modules/react-native-gesture-handler/src/components/Swipeable.tsx
@@ -164,13 +164,13 @@ export interface SwipeableProps
    * Style object for the container (`Animated.View`), for example to override
    * `overflow: 'hidden'`.
    */
-  containerStyle?: StyleProp<ViewStyle>;
+  containerStyle?: Animated.WithAnimatedObject<ViewStyle> | Animated.WithAnimatedArray<ViewStyle>;
 
   /**
    * Style object for the children container (`Animated.View`), for example to
    * apply `flex: 1`
    */
-  childrenContainerStyle?: StyleProp<ViewStyle>;
+  childrenContainerStyle?: Animated.WithAnimatedObject<ViewStyle> | Animated.WithAnimatedArray<ViewStyle>;
 }
 
 type SwipeableState = {
